### DISCUSSION TITLE
Creating Side Panels on Grafana:

### DISCUSSION THREAD

**Discussion by MeenakshiIOT** [grafana/grafana#70143] [Dashboards, Panels & Visualizations]
Is there a possibility to create side panels for the dashboard. If so yes, then how can we do that and if not then are there any plugins available supported by Grafana which can accomplish the same?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82644. Feel free to continue the discussion around this there. Thank you!
